client/src/stories/mainLogo.stories.js

```diff
@@ -0,0 +1,22 @@
+import { MainLogo } from "./mainLogo";
+import { Header } from "./mainLogo";
+import { HorizontalLine } from "./mainLogo";
+import { HeaderTwo } from "./mainLogo";
+
+import logo from "../stories/assets/icon/fabstr.png";
+
+export default {
+  title: "Component/MainLogo",
+  component: MainLogo,
+};
+
+export const Main_Logo = () => {
+  return (
+    <>
+      <MainLogo src={logo} alt="logo" />
+      <Header>Fabstr.</Header>
+      <HorizontalLine />
+      <HeaderTwo>Therapeutic Fasting</HeaderTwo>
```

Comment:
> -As soon as I change the font size of h2 the logo(img) is not centered anymore.

The Logo has got the same alignment as the h2, but h1 is centre aligned. Maybe change both(logo +h1) to center align or build a main-div with center-aligned content around it ?